client/lib/merkle_proofs.rs

```diff
@@ -152,3 +178,33 @@ pub(crate) fn validate_get_balance_response(
     )
     .map_err(Into::into)
 }
+
+pub(crate) fn validate_get_block_response(
```

Comment:
I don't really consider this as validating a merkle proof, so if you are going to add this here (and I think it probably is the right place for it) then I think we need to change the module name from `merkle_proofs` to `validation` or the like.

Reply:
Fixed in https://github.com/CasperLabs/casper-node/pull/456/commits/ccfccaaeb9cfc26874fe17c4b02b6abbce8f4870